endpoints/kaspad_requests/submit_transaction_request.py

```diff
@@ -0,0 +1,121 @@
+# encoding: utf-8
+import time
+from typing import List
+
+from pydantic import BaseModel
+
+from server import app, kaspad_client
+
+
+class TxOutpoint(BaseModel):
+    transactionId: str
+    index: int
+
+
+class TxInput(BaseModel):
+    previousOutpoint: TxOutpoint
+    signatureScript: str
+    sequence: int
+    sigOpCount: int
+
+
+class TxScriptPublicKey(BaseModel):
+    version: int
+    scriptPublicKey: str
+
+
+class TxOutputVerboseData(BaseModel):
+    scriptPublicKeyType: str
+    scriptPublicKeyAddress: str
+
+
+class TxOutput(BaseModel):
+    amount: int
+    scriptPublicKey: TxScriptPublicKey
+    # verboseData: TxOutputVerboseData | None
+
+
+class TxVerboseData(BaseModel):
+    transactionId: str
+    hash: str
+    mass: int
+    blockHash: str
+    blockTime: int
+
+
+class TxModel(BaseModel):
+    version: int
+    inputs: List[TxInput]
+    outputs: List[TxOutput]
+    lockTime: int | None = 0
+    subnetworkId: str | None
+
+
+class SubmitTransactionRequest(BaseModel):
+    transaction: TxModel
+    allowOrphan: bool = True
+
+
+class SubmitTransactionResponse(BaseModel):
+    transactionId: str | None
+    error: str | None
+
+
+@app.post("/transactions",
+          response_model=SubmitTransactionResponse,
+          tags=["Kaspa transactions"],
+          response_model_exclude_unset=True)
+async def submit_a_new_transaction(body: SubmitTransactionRequest):
+    """
+    Forwards the body directly to kaspad with the command submitTransactionRequest
+    """
+    tx_resp = await kaspad_client.request("submitTransactionRequest",
+                                          params=body.dict())
+
+    tx_resp = tx_resp["submitTransactionResponse"]
+    if "error" in tx_resp:
```

Comment:
Is the `error` property filled when this path of the RPC is called https://github.com/kaspanet/kaspad/blob/377d9aaaebd4a64199afa7f66372bdb2ea296858/app/rpc/rpchandlers/submit_transaction.go#L26 ?

I can see this `if` statement would handle the cases where:
- transaction couldn't be parsed
- transaction is rejected because it's already in the mempool

But I'm not sure if it will properly catch the 3rd case I linked above.